### DISCUSSION TITLE
Como começar no mercado de UX/UI?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1268] [Discussão]
A um tempo atrás eu abri uma ISSUE perguntando se era necessário faculdade para trabalhar com Front End. Porém, desde então meus rumos estão mudando e eu realmente vi que meu interesse é Design. Vou iniciar um curso tecnólogo de Design Gráfico ano que vem só pra pegar o diploma. Nesse tempo, eu comecei a me interessar muito por UX/UI Design. Como acompanho esse forum a um tempo, resolvi vir procurar algumas respostas por aqui, pois achei muitos poucos blogs sobre UX/UI. Só achei o UX Collective, mas é muito focado em UX. Alguém trabalha com UX/UI? Gostaria muito de saber qual caminho seguir e o que estudar. Tenho estudado os conceitos de design pra depois focar no que quero. E por mais que seja óbvia a pergunta o UI Designer TEM que saber HTML e CSS ou não é obrigatório?

**Comment by felipefialho** [comment]
Linkando algumas issues semelhantes:

#479 #415 #815 #1248

**Comment by mutheusalmeida** [comment]
Eu também sou iniciante e apenas recentemente percebi que eu gosto do Front-end focado em design.

Eu estou fazendo o currículo da freeCodeCamp, me inscrevi na mail list do Zell Liew, Matej Latin, CSS Weekly, Frontend Focus e sigo o Andrew Coyle e quem trabalha com isso no Twitter. Também comecei a praticar mais no CodePen e a estudar CSS grid, que é bastante importante. Se eu pudesse ser o tutor de alguém hoje, eu recomendaria praticar muito. A maioria dos iniciantes não sabem indentificar características fundamentais de uma tecnologia, leem bastante e não aplicam esse conhecimento em nada. Diferente do desenvolvedor mais experiente que sabe indentificar e procurou estudar justamente porque irá aplicar em algo. É "Eu preciso fazer um formulário responsivo, então eu vou estudar CSS grid". É uma questão psicológica.

E não é necessário ter faculdade. Isso é uma realidade para várias profissões. A maioria dos desenvolvedores não tem e simplesmente fazem isso.

Eu recomendo a plataforma Dribbble.

Sim, tem que saber HTML e CSS. E se especializar nessas tecnologias é um bom ramo.

Espero ter ajudado!

**Comment by bernardodiasc** [comment]
> UI Designer TEM que saber HTML 

Tem que saber HTML sim, muito mais que qualquer outra linguagem. A razão disso é que o profissional que desenha interface de usuário tem que ser capaz de entender muito de **conteúdo**.

Todos os detalhes técnicos de linha de código que voce vai ter que estudar muito pra aprender são importantes, mas nada se compara ao conteúdo. A função de um profissional de UI é traduzir uma história do cliente para o consumidor. É criar uma forma atraente, elegante, que seduza o usuário aos objetivos do dono do produto. Apesar de cores e formas serem parte importante dessa sedução, o conteúdo é a peça fundamental, e dominar semânticas (como um todo e não apenas a semântica do HTML) se torna um diferencial enorme entre quem é operador de photoshop pra alguém que domina o que está fazendo e entende como alcançar resultados melhores para os clientes.

Saber HTML e CSS vai te ajudar a criar sabendo como seriam as implementações. Isso pode causar um efeito reverso que já vi acontecer muitas vezes, onde por pouca experiência técnica o profissional acaba com sua criatividade restringida. Ao passo que também já vi designers sem conhecimento de código criando peças incríveis porém totalmente impraticáveis no browser.

Enfim, estude HTML e CSS sim, bastante, especialmente se seu foco for ser produzir para navegadores. Mas independentemente do tipo de material que voce irá produzir, se é pra navegadores ou qualquer outro mídia, seja digital ou impressa, lembre-se sempre: o conteúdo é o produto, o resto é perfumaria. E como ser bom em conteúdo? Leia e escreva! Leia muito sobre tudo. Leia muito sobre o cliente, sobre o projeto do cliente, sobre o ramo de atividade do cliente. Entreviste o cliente, os usuários, recolha informações das pessoas que voce conhece, escreva tudo e estude. Escreva muito também, sobre qualquer coisa.

**Comment by phenriqueleao** [comment]
Aqui tem uma aula sobre o assunto:

[https://www.youtube.com/watch?v=WAXrIApaHOQ](url)

**Comment by fylipeaguilar** [comment]
Alguém tem um ferramenta para criação de mocup de tela web free para indicar?

**Comment by ninetails** [comment]
As melhores ferramentas de mercado possuem free tier:
- [Figma](https://www.figma.com/pricing/) - 3 projetos 2 editores
- [Zeplin](https://zeplin.io/pricing) - 1 projeto
- [Adobe XD](https://www.adobe.com/br/products/xd/compare-plans.html) - 1 projeto
- [Invision](https://www.invisionapp.com/plans) - 1 projeto
- [Marvel](https://marvelapp.com/pricing) - 1 usuário 1 projeto

Aquela coisa, se é pra uso pessoal ou pra quem tá começando, de boas. Mas cogite pagar caso for usar profissionalmente pq as ferramentas são boas :D

Não testei mas:
- [Origami](https://origami.design/) - offline app
- [Mockplus](https://www.mockplus.com/buy/mockplus-idoc) - 10 projetos 5 membros

**Comment by fylipeaguilar** [comment]
@ninetails,

Entendi que todas (após "x" projetos) acabam sendo pagas. Free você não conhece nenhuma?

**Comment by ninetails** [comment]
@fylipeaguilar então, pra indicar, não.

Mas para qual será o uso? A maioria das ferramentas que citei, para uso pessoal difícil acabar precisando usar mais de um projeto ao mesmo tempo pois se não me engano a maioria possui funcionalidade para arquivar projeto que vc não esteja ativamente utilizando, não contando como projeto ativo. Saindo disso, creio que a utilização possa ser considerada mais profissional, ou seja, vale pagar e vc pode imbutir isto no preço do teu serviço, difícil ferramenta que possibilite ser acessado online ser free. Já se for para uso educacional, não cheguei a pesquisar a fundo cada ferramenta.

A única ferramenta que vi open source seria o [Pencil](http://pencil.evolus.vn/). Dando uma googlada, achei [esse tb](https://meetalva.io/). Ainda sem hosting, pode ser até que exista alguma integrada com Google Docs.

**Comment by fylipeaguilar** [comment]
Obrigadão, @ninetails!!
